src/Lykke.Service.BlockchainCashoutPreconditionsCheck/Modules/ServiceModule.cs

```diff
@@ -12,13 +13,16 @@ namespace Lykke.Service.BlockchainCashoutPreconditionsCheck.Modules
     public class ServiceModule : Module
     {
         private readonly IReloadingManager<BlockchainCashoutPreconditionsCheckSettings> _settings;
+
+        private readonly IReloadingManager<BlockchainWalletsServiceClientSettings> _walletClientSettings;
```

Comment:
IReloadingManager is not necessary here, since BlockchainWalletsServiceClientSettings has no Azure Storage connection strings